src/translations/pl.json

```diff
@@ -0,0 +1,257 @@
+    "verified": "Zweryfikowano",
+    "verifiedSuccessfully": "Pomyślnie zweryfikowany",
+    "verify": "Zweryfikuj",
+    "verifyUser": "Zweryfikuj użytkownika",
+    "youAreInactive": "Nie byłeś aktywny od dłuższego czasu i wkrótce zostaniesz automatycznie wylogowany dla własnego bezpieczeństwa. Czy chcesz pozostać zalogowany?",
+    "youAreReceivingResetPassword": "Otrzymałeś tę wiadomość, ponieważ Ty (lub ktoś inny) poprosiłeś o zresetowanie hasła do Twojego konta. Kliknij poniższy link lub wklej go w przeglądarce, aby zakończyć proces:",
+    "youDidNotRequestPassword": "Jeśli nie prosiłeś o zmianę hasła, zignoruj tę wiadomość, a Twoje hasło pozostanie niezmienione."
+  },
+  "error": {
+    "accountAlreadyActivated": "To konto zostało już aktywowane.",
+    "autosaving": "Wystąpił problem podczas automatycznego zapisywania tego dokumentu.",
+    "correctInvalidFields": "Popraw nieprawidłowe pola.",
+    "deletingTitle": "Wystąpił błąd podczas usuwania {{title}}. Proszę, sprawdź swoje połączenie i spróbuj ponownie.",
+    "emailOrPasswordIncorrect": "Podany adres e-mail lub hasło jest nieprawidłowe.",
+    "followingFieldsInvalid_many": "Następujące pola są nieprawidłowe:",
+    "followingFieldsInvalid_one": "To pole jest nieprawidłowe:",
+    "forbidden": "Nie możesz wykonać tej akcji.",
+    "incorrectCollection": "Nieprawidłowa kolekcja",
+    "loadingDocument": "Wystapił problem podczas ładowania dokumentu o ID {{id}}.",
+    "missingEmail": "Brak adresu email.",
+    "missingIDOfDocument": "Brak ID dokumentu do aktualizacji.",
+    "missingIDOfVersion": "Brak ID wersji",
+    "missingRequiredData": "Brak wymaganych danych.",
+    "noFilesUploaded": "Nie przesłano żadnych plików.",
+    "noMatchedField": "Nie znaleziono pasującego pola dla \"{{label}}\"",
+    "noUser": "Brak użytkownika",
+    "notFound": "Żądany zasób nie został znaleziony.",
+    "problemUploadingFile": "Wystąpił problem podczas przesyłania pliku.",
+    "tokenInvalidOrExpired": "Token jest nieprawidłowy lub wygasł.",
+    "unPublishingDocument": "Wystąpił problem podczas cofania publikacji tego dokumentu.",
+    "unauthorized": "Brak dostępu, musisz być zalogowany.",
+    "unknown": "Wystapił nieznany błąd.",
+    "unspecific": "Wystąpił błąd",
+    "userLocked": "Ten użytkownik został zablokowany z powodu zbyt wielu nieudanych prób logowania.",
+    "verificationTokenInvalid": "Token weryfikacyjny jest nieprawidłowy."
+  },
+  "fields": {
+    "Block": "Blok",
+    "Blocks": "Bloki",
+    "addLabel": "Dodaj {{label}}",
+    "addNew": "Dodaj nowy",
+    "addNewLabel": "Dodaj nowy {{label}}",
+    "addRelationship": "Dodaj relacje",
+    "blockType": "Typ Bloku",
+    "chooseFromExisting": "Wybierz z istniejących",
+    "chooseLabel": "Wybierz {{label}}",
+    "collapseAll": "Zwiń wszystko",
+    "editLabelData": "Edytuj dane {{label}}",
+    "fieldHasNo": "To pole nie posiada {{label}}",
+    "fieldRequiresRows": "To pole wymaga conajmniej {{label}}",
```

Comment:
`co najmniej`

Reply:
Shame on me. My polish teacher would be disappointed. Luckily she doesn't have an account on GH. :D